experimental/auxiliary_sampling/run_training.py

```diff
@@ -292,59 +262,52 @@ def fit_fn(neural_net,
           # Fix the variance of the prior
           session.run(l.bias_prior.distribution.istrainable.assign(0.))
 
-    # Ensemble training
     ensemble_filenames = []
-    for i in range(FLAGS.n_ensemble):
-      neural_net.load_weights(base_model_filename)
+    for i in range(FLAGS.ensemble_size):
+      model.load_weights(base_model_filename)
```

Comment:
Loading the weight can be very slow, this is something we want to keep in mind when measuring the runtime.

Reply:
SGTM